app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MenusController < ApplicationController
+  before_action :set_menu, only: [:show, :edit, :update]
+  load_and_authorize_resource
+
+  def index
+    @menus = Menu.all
+  end
+
+  def show
+  end
+
+  def edit
+    redirect_to menus_path unless @menu.for_today?
+  end
+
+  def update
+    if @menu.update(menu_params)
```

Comment:
`return redirect_to menus_path if @menu.update(menu_params)`
my personal issues :) but as I recall, we always preferred inliners + this thing with `and return` is already forgotten and killed